packages/core/src/client.ts

```diff
@@ -113,6 +114,15 @@ export interface MedplumClientOptions {
    */
   cacheTime?: number;
 
+  /**
+   * The length of time in milliseconds to delay requests for auto batching.
+   *
+   * Auto batching attempts to group multiple requests together into a single batch request.
+   *
+   * Default value is 0, which disables auto batching.
+   */
+  autoBatchTime?: number;
```

Comment:
One small request - can we get a getter/setter for autoBatchTime? As a component developer, I might want to enable autobatching for a specific component, and then disable again without creating a new client. 

But i'm not sure if that breaks with concurrent rendering